reconcile/saas_file_owners.py

```diff
@@ -268,6 +268,11 @@ def run(dry_run, gitlab_project_id=None, gitlab_merge_request_id=None,
     valid_saas_file_changes_only = is_saas_file_changes_only and is_valid_diff
     write_diffs_to_file(io_dir, diffs, valid_saas_file_changes_only)
 
+    # print 'yes' or 'no' to allow pr-check to understand if changes
+    # are only valid saas file changes (and exclude other integrations)
+    output = 'yes' if valid_saas_file_changes_only else 'no'
+    print(output)
```

Comment:
I'm assumning this is temporary? If not, please use `logging`.

Reply:
this is actually the intention: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/10500/diffs#103cbab4c3108c4f7f19aefb17f404405aaae36c_117_116

Reply:
yep, logging ftw